microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/core/ls/MicroProfileDelegateCommandHandlerForJava.java

```diff
@@ -39,6 +47,7 @@
 
 	private static final String JAVA_CODELENS_COMMAND_ID = "microprofile/java/codeLens";
 	private static final String JAVA_HOVER_COMMAND_ID = "microprofile/java/hover";
+	private static final String JAVA_PROJECT_TYPE_COMMAND_ID = "microprofile/java/projectTypes";
```

Comment:
I think it should be better to create a new delegate command handler like `MicroProfileDelegateCommandHandlerForJavaProject` because the idea is that MicroProfileDelegateCommandHandler / MicroProfileDelegateCommandHandlerForJava are used for MicroProfile LS (application.properties / Java files), your new command is not used by MicroProfile LS